libtrellis/tools/ecpunpack.cpp

```diff
@@ -15,7 +15,7 @@ int main(int argc, char *argv[])
     using namespace Trellis;
     namespace po = boost::program_options;
 
-    std::string database_folder = TRELLIS_PREFIX "/share/trellis/database";
+    std::string database_folder = TRELLIS_PREFIX "/share/prjtrellis/database";
```

Comment:
I'd prefer to keep this path as trellis, now that many people have it installed already. Otherwise people will end up with two copies of everything which will be confusing.